ecl/network/v2/qos_options/testing/fixtures.go

```diff
@@ -0,0 +1,95 @@
+package testing
+
+import "github.com/nttcom/eclcloud/ecl/network/v2/qos_options"
+
+const ListResponse = `
+{
+	"qos_options": [
+	  	{
+			  "aws_service_id"		: "3472d029-8378-424c-aaaf-150edc08947b",
+			  "azure_service_id"	: "",
```

Comment:
API Doc does not show blank string but null. The same applies to the other IDs. Please confirm which is correct.
https://ecl.ntt.com/en/documents/api-references/network/azure/qos_option.html#sample-response-body